TESTING.md

```diff
@@ -0,0 +1,59 @@
+# Testing CP-Ansible
+
+Each role has Molecule configuration which enable fast and easy development and testing with Docker containers.
```

Comment:
Grammar and spelling.  "Each role has a Molecule configuration which enables fast and easy development, including testing.  Molecule takes advantage of building out an environment using Docker.

We should also outline the details, or link to the document to setup Molecule.

Reply:
I would also embed the link to the official molecule docs in the "Molecule" word in the paragraph.  And under each sub section link to the relevant docs for molecule.